lib/data/gmailSupportEmail.ts

```
@@ -40,7 +40,13 @@ export const createGmailSupportEmail = async (
     .values({
       email: info.email,
       ...("accessToken" in info
-        ? { accessToken: info.accessToken, refreshToken: info.refreshToken, expiresAt: info.expiresAt }
+        ? { 
```

ESLint

[error] Delete `·`

(prettier/prettier)